### DISCUSSION TITLE
Sign in not working, infinite pending

### DISCUSSION THREAD

**Discussion by joshuavachon25** [supabase/supabase#8014] [Questions]
I created a new project with Sveltekit and Supabase. I used an other (working) project as template and I just changed the Supabase keys. The problem is, when I try to login, it doesn't work. In the network tab, I can see the request to token?grant_type=password; the preflight returns 200, but the fetch never returns the response and stays in a 'pending' status. When I check the payload of the fetch, I can see the email and the password, plus headers contains the API key, so I really don't understand what is going on. I even tried to restart Supabase instance, but it did not change anything.

Can someone help me out with that please?

**Comment by dshukertjr** [comment]
Hi @joshuavachon25,

Would you be able to provide the code sample that you are trying to make it work so that our Sveltkit expert @silentworks can take a look?

**Reply by joshuavachon25** [reply]
After analysis, the problem is not from the code, but from the Supabase instance. If I just replace the URL and Key for any other project, the are no error.

The instance was automatically paused last week for inactivity and I think the problem emerge from that. The instance is not paused anymore, but it feels like a part of it still is... 

To solve my problem for now, I created another Supabase instance and it works. I did not delete the other instance, because I think that the bug could remain a problem for other users. So, it's still possible to verify if the bug is corrected.

**Reply by dshukertjr** [reply]
I see. If you don't mind, will you be able to email `support@supabase.io` with the project reference of the project with the issue and we can take a look at what exactly was the problem there.
